test/finch_test.exs

```diff
@@ -20,21 +20,33 @@ defmodule FinchTest do
       start_supervised({Finch, name: MyFinch})
       expect_any(bypass)
 
-      {:ok, %Response{}} = Finch.request(MyFinch, :get, endpoint(bypass), [], "")
+      {:ok, %Response{}} = Finch.request(MyFinch, :get, endpoint(bypass))
       assert [_pool] = get_pools(MyFinch, shp(bypass))
 
-      {:ok, %Response{}} = Finch.request(MyFinch, :get, endpoint(bypass), [], "")
+      {:ok, %Response{}} = Finch.request(MyFinch, :get, endpoint(bypass))
     end
 
     test "default can be configured", %{bypass: bypass} do
-      {:ok, _} = Finch.start_link(name: MyFinch, pools: %{default: %{count: 5, size: 5}})
+      {:ok, _} =
+        Finch.start_link(
+          name: MyFinch,
+          pools: %{default: [count: 5, size: 5, backoff: [size: 2, count: 4]]}
```

Comment:
Is the `backoff` configuration here intentional? I'm not sure I'm following exactly what's happening in this test.

Reply:
The values are arbitrary, and I added the configuration here, just to show that the default can be configured with all available options. But since I am not asserting that the values were actually set, I will remove them from here, and properly test backoff configuration after we merge the telemetry pr.

Reply:
Ah gotcha :) Thanks for the explanation

Reply:
Doh, you meant why did I use size & count instead of initial & max, that was a complete accident! Good news is that NimbleOptions made that clear immediately 🙃